docs/serving/samples/hello-world/helloworld-shell/index.md

```diff
@@ -5,4 +5,206 @@ weight: 1
 type: "docs"
 ---
 
-{{% readfile file="README.md" %}}
+This guide describes the steps required to create the `helloworld-go` sample app and deploy it to your
+cluster.
+
+The sample app reads a `TARGET` environment variable, and prints `Hello ${TARGET}!`.
+If `TARGET` is not specified, `World` is used as the default value.
+
+You can also download a working copy of the sample, by running the
+following commands:
+
+```shell
+git clone -b "{{< branch >}}" https://github.com/knative/docs knative-docs
+cd knative-docs/docs/serving/samples/hello-world/helloworld-shell
+```
+
+## Prerequisites
+
+- A Kubernetes cluster with Knative installed and DNS configured. Follow the
+  [installation instructions](../../../../install/README.md).
+- [Docker](https://www.docker.com) installed and running on your local machine,
+  and a Docker Hub account configured.
```

Comment:
Since Github supports image registry with [GitHub Docker packages](https://github.com/maximilien?package_type=Docker&tab=packages) I wonder if we should also include that here since it keeps everything tidy with GH. Thoughts?

Reply:
I'd stick with DockerHub for now.  If we switch to GHCR we should switch everything, I think.